### PR TITLE
CenterDetails component

### PR DESCRIPTION
**Pull request by OlegGre4ka**
Todo:
- [ ] - created CenterDetails component - layout,display data;
- [ ] - saving data with Mobx(persist into local storage) for reloading CenterDetails page

### CODE REVIEW

File: web-client/src/view/BloodCenters/index.tsx [hunk 1]
```diff
@@ -141,14 +148,22 @@ const BloodCenters: React.FC = observer(() => {
...
+            <Card key={center.id} onClick={()=>navigate(`center/${center.id}`)} cursor="pointer">
+              <FlexBox align="center">
+                <LocationIcon />
+                <FlexBox direction="column"  margin="0 0 0 16px">
+                  <Text color="var(--header-text-color)" fontWeight="600" cursor="pointer">{center.name}</Text>
+                  <Text color="var(--header-text-color)" cursor="pointer">{center?.address?.street}, {center?.address?.city}</Text>
+                  <Text color="var(--header-text-color)" cursor="pointer">{center?.address?.postalCode}, {center?.address?.region}</Text>
+                </FlexBox>
+              </FlexBox>
+              <Badge>{transformFirstLetter(center.category)}</Badge>
```
Comment (valoricDe) [inline web-client/src/view/BloodCenters/index.tsx:162]: Would css text-transform: capitalize; also work?

File: web-client/src/view/BloodCenters/CenterDetails/index.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, { useState, useEffect } from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
+import { IBloodDonationCenter } from 'interfaces/bloodDonationCenters';
+import { Card, LocationIcon } from './../styled';
+import Breadcrumbs from '@mui/material/Breadcrumbs';
+// import Link from '@mui/material/Link';
```
Comment (valoricDe) [inline web-client/src/view/BloodCenters/CenterDetails/index.tsx:7]: cleanup possible :)

File: web-client/src/view/BloodCenters/CenterDetails/index.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+import bloodCentersStore from 'store/bloodCenters';
+// import FileCopyIcon from '@mui/icons-material/FileCopy';
+import PhoneIcon from '@mui/icons-material/Phone';
+// import MailIcon from '@mui/icons-material/Mail';
+import AlternateEmailIcon from '@mui/icons-material/AlternateEmail';
+import PublicIcon from '@mui/icons-material/Public';
+// import Link from '@mui/material/Link';
+import Link from 'components/StyledComponents/Link';
+import { CopyIcon } from 'components/Icons';
+const CenterDetails: React.FC = () => {
```
Comment (valoricDe) [inline web-client/src/view/BloodCenters/CenterDetails/index.tsx:19]: newline

**Review by valoricDe — APPROVED** [review]

File: web-client/src/view/BloodCenters/CenterDetails/index.tsx (hunk 2 above)
Comment (OlegGre4ka) [reply web-client/src/view/BloodCenters/CenterDetails/index.tsx:7]: +

File: web-client/src/view/BloodCenters/CenterDetails/index.tsx (hunk 3 above)
Comment (OlegGre4ka) [reply web-client/src/view/BloodCenters/CenterDetails/index.tsx:19]: done

File: web-client/src/view/BloodCenters/index.tsx (hunk 1 above)
Comment (OlegGre4ka) [reply web-client/src/view/BloodCenters/index.tsx:162]: We receive all the letters capitalized, so I needed to create this helper. But I thank you for this question)! I found another solution:
  text-transform: lowercase;
  &:first-letter {
    text-transform: uppercase;
  }

**Pull request merged**